main/Button/index.tsx

```diff
@@ -105,8 +93,7 @@ function Button(props: Props): React.ReactElement {
       onPress={onPress}
       style={containerStyle}
       delayPressIn={30}
-      {...touchableOpacityProps}
-    >
+      {...touchableOpacityProps}>
```

Comment:
I'd prefer a new line here. However, I'll just merge this time since it doesn't look critical.